src/test/kotlin/graphics/scenery/tests/unit/backends/FauxRenderer.kt

```diff
@@ -87,6 +87,7 @@ class FauxRenderer(override var hub: Hub?, var scene: Scene, preparedWindow: Sce
     override var renderConfigFile: String = ""
 
     private var recordMovie = false
+
```

Comment:
Likely a spurious space, right? If yes, remove

Reply:
a19f38f0ed029a8156bd2f602c4ff9fc234379d3 removes the spurious space